### PR TITLE
HOLD - Northern tribes2

### PR DESCRIPTION
**Pull request by featherus**

### CODE REVIEW

File: events/GriffonRepublic Events.txt [hunk 1]
```diff
@@ -1,21 +1,5 @@
 ﻿add_namespace = griffonRepublic
-add_namespace = kemerskaiSpeech
```
Comment (featherus) [inline events/GriffonRepublic Events.txt:2]: Why these changes? @jihaslun please review.

File: common/units/equipment/convoys.txt [hunk 2]
```diff
@@ -54,4 +54,35 @@ equipments = {
 		active = yes
 		priority = 10
 	}
+	
+	icebreaker = {
```
Comment (featherus) [inline common/units/equipment/convoys.txt:58]: Why does this exist? It will need testing.

File: common/technology_sharing/eaw_tech_sharing_groups.txt [hunk 3]
```diff
@@ -1,23 +1,16 @@
...
+	
 	research_sharing_per_country_bonus = 0.1
-
+	
 	categories = { industry }
-
+	
 	available = {
 		has_government = democratic
-		NOT = { has_war_with = event_target:equestria }
```
Comment (featherus) [inline common/technology_sharing/eaw_tech_sharing_groups.txt:14]: Why did you remove this?

File: common/technology_sharing/eaw_tech_sharing_groups.txt [hunk 4]
```diff
@@ -51,24 +44,17 @@ technology_sharing_group = {
...
-
+	
 	research_sharing_per_country_bonus = 0.1
-
+	
 	categories = { land_doctrine }
-
+	
 	available = {
-		NOT = { has_war_with = CHN }
```
Comment (featherus) [inline common/technology_sharing/eaw_tech_sharing_groups.txt:65]: Why did you remove this?

File: common/technology_sharing/eaw_tech_sharing_groups.txt [hunk 5]
```diff
@@ -80,22 +66,15 @@ technology_sharing_group = {
...
 	name = CHN_changeling_advisor_in_PLB
 	desc = CHN_changeling_advisor_in_PLB_desc
 	picture = GFX_technology_sharing_default
-
+	
 	research_sharing_per_country_bonus = 0.1
-
+	
 	available = {
-		NOT = { has_war_with = CHN }
```
Comment (featherus) [inline common/technology_sharing/eaw_tech_sharing_groups.txt:92]: Why did you remove this?

File: common/technology_sharing/eaw_tech_sharing_groups.txt [hunk 6]
```diff
@@ -142,144 +121,116 @@ technology_sharing_group = {
...
 		is_in_faction_with = GRF
-		GRF = {
-			is_faction_leader = yes
-		}
-	}
-}
-
-technology_sharing_group = {
-
-	id = CRY_SOL_magic_tech_share
```
Comment (featherus) [inline common/technology_sharing/eaw_tech_sharing_groups.txt:250]: Why did you remove this?

File: common/technologies/infantry.txt [hunk 7]
```diff
@@ -163,7 +163,7 @@ technologies = {
 		}
 
 		ai_will_do = {
-			factor = 1000000
+			factor = 1000
```
Comment (featherus) [inline common/technologies/infantry.txt:166]: Why? Has this been balance approved to see its effect?

File: common/technologies/air_doctrine.txt [hunk 8]
```diff
@@ -1465,6 +1464,10 @@ technologies = {
 				ai_land_doctrine_tier_2_trigger = yes
 				factor = 5
 			}
+			modifier = {
+				factor = 0
+				tag = SCS
```
Comment (featherus) [inline common/technologies/air_doctrine.txt:1469]: Has this been approved by @Jas-PY?

File: common/scripted_triggers/EAW_triggers.txt [hunk 9]
```diff
@@ -1391,81 +1398,6 @@ is_a_potential_enemy_or_has_war_with_ROOT = {
 	}
 }
 
-has_dangerous_naval_war = {
```
Comment (featherus) [inline common/scripted_triggers/EAW_triggers.txt:1394]: Why did you remove these?

File: common/decisions/SRS_decisions.txt [hunk 10]
```diff
@@ -1,12 +1,8 @@
 SRS_political_discussion = {
 	SRS_elections_timer = {
 		icon = eng_propaganda_campaigns
-		activation = {
```
Comment (featherus) [inline common/decisions/SRS_decisions.txt:4]: Why all these changes? Please consult @Jas-PY

File: common/ideas/_economic.txt [hunk 11]
```diff
@@ -14,46 +13,46 @@ ideas = {
 					limit = { has_country_flag = GRI_junta_established }
 					NOT = { has_completed_focus = GRI_the_war_economy }
 				}
-				if = {
+				else_if = {
 					limit = { has_completed_focus = FEA_hermans_stratocracy }
 					NOT = { has_completed_focus = FEA_retool_the_industrial_sector }
 				}
-				if = {
+				else_if = {
```
Comment (featherus) [inline common/ideas/_economic.txt:20]: These should not be else ifs

File: common/ideas/_economic.txt [hunk 12]
```diff
@@ -63,225 +62,226 @@ ideas = {
 					limit = { has_country_flag = GRI_junta_established }
 					NOT = { has_completed_focus = GRI_the_war_economy }
 				}
-				if = {
+				else_if = {
```
Comment (featherus) [inline common/ideas/_economic.txt:65]: These should not be else ifs

File: common/ideas/_economic.txt [hunk 13]
```diff
@@ -63,225 +62,226 @@ ideas = {
...
 		}
 		
 		outlaw_economy = {
             picture = outlaw_economy
+			
 			allowed = {
 				always = no
 			}
-
-			cost = 150
```
Comment (featherus) [inline common/ideas/_economic.txt:138]: Please don't change this

File: common/ideas/_economic.txt [hunk 14]
```diff
@@ -63,225 +62,226 @@ ideas = {
...
-			    if = {
-					limit = { has_country_flag = GRI_junta_established }
-					NOT = { has_completed_focus = GRI_the_war_economy }
-				}
 				if = {
 					limit = { has_completed_focus = FEA_hermans_stratocracy }
 					NOT = { has_completed_focus = FEA_retool_the_industrial_sector }
 				}
-				if = {
+				else_if = {
```
Comment (featherus) [inline common/ideas/_economic.txt:211]: These should not be else ifs

File: common/ideas/_economic.txt [hunk 15]
```diff
@@ -63,225 +62,226 @@ ideas = {
...
-					limit = { has_country_flag = GRI_junta_established }
-					NOT = { has_completed_focus = GRI_the_war_economy }
-				}
-				if = {
+				else_if = {
 					limit = { has_completed_focus = FEA_hermans_stratocracy }
 					NOT = { has_completed_focus = FEA_retool_the_industrial_sector }
 				}
-				if = {
+				else_if = {
```
Comment (featherus) [inline common/ideas/_economic.txt:276]: Ditto

File: common/ideas/_economic.txt [hunk 16]
```diff
@@ -293,104 +293,83 @@ ideas = {
...
 		war_economy = {
-
-
 			available = {
 				if = {
 					limit = { has_country_flag = is_maresoc }
 					has_country_flag = is_maresoc
 				}
-				if = {
+				else_if = {
```
Comment (featherus) [inline common/ideas/_economic.txt:332]: Not an else if

File: common/ideas/_economic.txt [hunk 17]
```diff
@@ -420,25 +397,21 @@ ideas = {
...
+						surrender_progress > 0.5
 						any_enemy_country = {
 							ic_ratio = {
 								tag = ROOT
 								ratio > 0.5
 							}
 						}
-						surrender_progress > 0.5
 					}
-
```
Comment (featherus) [inline common/ideas/_economic.txt:431]: No to these changes

File: common/ideas/_economic.txt [hunk 18]
```diff
@@ -448,140 +421,132 @@ ideas = {
...
-					has_country_flag = demilitarized
-				}
-				NOT = { has_country_flag = economic_laws_blocked }
-				NOT = {
-					has_country_flag = BAL_anarchy_laws
-				}
 			}
-
+			
+			modifier = {
```
Comment (featherus) [inline common/ideas/_economic.txt:426]: Please don't do whatever these changes here are.

File: common/ideas/_economic.txt [hunk 19]
```diff
@@ -448,140 +421,132 @@ ideas = {
...
 					factor = 0
+					
 					OR = {
 						has_idea = volunteer_only
 						has_idea = limited_conscription
 					}
 				}
 			}
-
-			cost = 150
```
Comment (featherus) [inline common/ideas/_economic.txt:486]: Why Remove?!

File: common/ideas/_economic.txt [hunk 20]
```diff
@@ -448,140 +421,132 @@ ideas = {
...
 				}
 			}
 			
 			available = {
-				NOT = { has_country_flag = trade_laws_blocked }
 				if = {
 					limit = { has_idea = EQC_SOL_autarky_idea }
 					NOT = { has_idea = EQC_SOL_autarky_idea }
 				}
-				NOT = { has_country_flag = FLO_forced_limited }
```
Comment (featherus) [inline common/ideas/_economic.txt:523]: Why remove?

File: common/ideas/_economic.txt [hunk 21]
```diff
@@ -448,140 +421,132 @@ ideas = {
...
 					NOT = { has_completed_focus = FEA_state_controlled_economy }
 				}
+				else = {
+					NOT = {
+						has_country_flag = trade_laws_blocked
+						has_country_flag = FLO_forced_limited
+					}
+				}
 			}
-
```
Comment (featherus) [inline common/ideas/_economic.txt:538]: This should stay

File: common/ideas/_economic.txt [hunk 22]
```diff
@@ -448,140 +421,132 @@ ideas = {
...
+			
 			cancel_if_invalid = no
+			
+			removal_cost = -1
 		}
-
+		
 		export_focus = {
-			available = {
```
Comment (featherus) [inline common/ideas/_economic.txt:557]: Do not remove these

File: common/ideas/_economic.txt [hunk 23]
```diff
@@ -591,54 +556,46 @@ ideas = {
...
 			cancel_if_invalid = no
+			
+			removal_cost = -1
 		}
-
+		
 		limited_exports = {
-
 			available = {
-				OR = {
```
Comment (featherus) [inline common/ideas/_economic.txt:603]: Why change this?

File: common/ideas/_economic.txt [hunk 24]
```diff
@@ -694,41 +660,23 @@ ideas = {
 					has_idea = tot_economic_mobilisation
 				}
 				NOT = { has_country_flag = TRD_forced_market }
-				if = {
```
Comment (featherus) [inline common/ideas/_economic.txt:697]: Do not remove

File: common/ideas/_society_development_ideas.txt [hunk 25]
```diff
@@ -1,62 +1,115 @@
+# New Society Development Trigger
```
Comment (featherus) [inline common/ideas/_society_development_ideas.txt:1]: @Yard1 please investigate this.

File: common/modifiers/00_static_modifiers.txt [hunk 26]
```diff
@@ -257,15 +268,6 @@ ROM_military_modernization = {
 	license_tech_difference_speed = 0.25 #less cost increase for each tech level between the license and own tech level 
 }
 
-free_license = {
```
Comment (featherus) [inline common/modifiers/00_static_modifiers.txt:260]: Why is this removed?

File: common/national_focus/SRS.txt [hunk 27]
```diff
@@ -1737,54 +1876,71 @@ shared_focus = {
...
 		log = "[GetDateText]: [Root.GetName]: Focus SRS_war_1"
+		
 		custom_effect_tooltip = SRS_communalist_delegates_hold_a_majority_tt
 		effect_tooltip = { add_timed_idea = { days = 650 idea = SRS_weapons_for_the_front } }
+		
 		custom_effect_tooltip = SRS_internationalist_delegates_hold_a_majority_tt
 		effect_tooltip = { add_timed_idea = { days = 650 idea = SRS_defend_our_homes } }
-		hidden_effect = { 
-			if = {
+
```
Comment (featherus) [inline common/national_focus/SRS.txt:1917]: @Jas-PY please review these changes

File: common/on_actions/eaw_on_actions.txt [hunk 28]
```diff
@@ -5,10 +5,26 @@ on_actions = {
 		effect = {
 			log = "[GetDateText] Equestria at War main on_startup begin"
 			every_state = {
-				limit = {
-					is_on_continent = europe #Equestria
+				if = {
+					limit = { in_dry_subarctic_climate_region = yes }
+					set_variable = { EaW_state_winter_grade_var = 5 }
```
Comment (featherus) [inline common/on_actions/eaw_on_actions.txt:10]: We are going to have to discuss these changes. What they are. And what impact they have on the game.

File: common/on_actions/eaw_on_actions.txt [hunk 29]
```diff
@@ -1719,7 +1926,34 @@ on_actions = {
 	on_release_as_puppet = {
 		effect = {
 			log = "[GetDateText]: ROOT: [Root.GetName] FROM: [From.GetName] on_release_as_puppet"
+
+			#if = {
```
Comment (featherus) [inline common/on_actions/eaw_on_actions.txt:1930]: Remove if not used

File: common/on_actions/eaw_on_actions.txt [hunk 30]
```diff
@@ -2655,55 +2956,156 @@ on_actions = {
...
 				limit = {
 					original_tag = FIR
-					FROM = { original_tag = FIR }
 				}
 				country_event = { id = FIR.29 days = 1 }
 			}
-		}
-	}
-	# ROOT is capitulated country, FROM is winner
-	on_capitulation = {
```
Comment (featherus) [inline common/on_actions/eaw_on_actions.txt:2693]: No. No. No. Whatever removing this is for. No.

File: common/on_actions/eaw_on_actions.txt [hunk 31]
```diff
@@ -2962,11 +3402,40 @@ on_actions = {
 			handle_twilight_escaping = yes
 		}
 	}
+
 	#ROOT is free nation FROM is releaser
 	on_release_as_free = {
 		effect = {
 			log = "[GetDateText]: ROOT: [Root.GetName] FROM: [From.GetName] on_release_as_free"
+
+			#if = {
```
Comment (featherus) [inline common/on_actions/eaw_on_actions.txt:3411]: Remove if not used

File: common/on_actions/eaw_on_actions.txt [hunk 32]
```diff
@@ -3799,188 +4287,2316 @@ on_actions = {
...
-								    NOT = { country_exists = VIN }
-									VIN = { is_subject_of = JER }
-                                }
-                                OR = {
-								    NOT = { country_exists = JFT }
-									JFT = { is_subject_of = JER }
-                                }
-								FROM = { tag = JEB }
+						}
+					}
```
Comment (featherus) [inline common/on_actions/eaw_on_actions.txt:4291]: WHY!?

File: common/opinion_modifiers/eaw_opinion_modifiers.txt [hunk 33]
```diff
@@ -1,6 +1,6 @@
-
-# OPINION MODIFIERS
```
Comment (featherus) [inline common/opinion_modifiers/eaw_opinion_modifiers.txt:2]: I am just saying no to your removals here

File: common/continuous_focus/generic.txt [hunk 34]
```diff
@@ -1,66 +1,70 @@
...
-		available = {
-		}
+		daily_cost = 1
+		available_if_capitulated = yes
 		
 		idea = continuous_naval_production
 		
 		ai_will_do = {
 			factor = 1
+
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/continuous_focus/generic.txt:24]: There used to be a chance to get stuck for a short while with 10+ completed focuses, but now it is gone, so this should be removed.

File: common/country_leader/00_traits.txt [hunk 35]
```diff
@@ -1871,6 +1883,22 @@ leader_traits = {
 		}
 	}
 	
+	popular_figurehead2 = {
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/country_leader/00_traits.txt:1886]: Didn’t notice, I will remove it.

File: common/technologies/air_doctrine.txt (hunk 8 above)
Comment (yastrom) [reply common/technologies/air_doctrine.txt:1469]: This is a duplicate, apparently I forgot to remove it.

File: common/decisions/SRS_decisions.txt (hunk 10 above)
Comment (yastrom) [reply common/decisions/SRS_decisions.txt:4]: All of these checks are already checked in the category, so they are not needed here.

File: common/ideas/_economic.txt (hunk 11 above)
Comment (yastrom) [reply common/ideas/_economic.txt:20]: Are you sure? Both focuses are in mutually exclusive trees, so how can they be completed at the same time?

File: common/on_actions/eaw_on_actions.txt (hunk 28 above)
Comment (yastrom) [reply common/on_actions/eaw_on_actions.txt:10]: Trying to understand the impact of broken mechanics should be fun.

File: common/on_actions/eaw_on_actions.txt (hunk 29 above)
Comment (yastrom) [reply common/on_actions/eaw_on_actions.txt:1930]: OK

File: common/on_actions/eaw_on_actions.txt (hunk 30 above)
Comment (yastrom) [reply common/on_actions/eaw_on_actions.txt:2693]: Take a closer look, just because of the large number of changes before this, git incorrectly selects brackets, although in fact the entire effect was moved lower.

File: common/decisions/MNC_decisions.txt [hunk 36]
```diff
@@ -128,17 +128,12 @@ MNC_expansion_category = {
...
-			NOT = {
-				has_country_flag = MNC_defeat_in_border_war
-			}
+			NOT = { has_country_flag = MNC_defeat_in_border_war }
 		}
 
 		visible = {
+			NOT = { owns_state = 574 }
 			has_war = no
-			has_completed_focus = MNC_elf_lake_expansion
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/decisions/MNC_decisions.txt:138]: mistake

File: common/decisions/_generic_decisions.txt [hunk 37]
```diff
@@ -77,28 +77,22 @@ political_actions = {
...
-			    if = {
-			        limit = { original_tag = TRD }
-					AND = {
-			            has_government = democratic
-				        has_completed_focus = TRD_altesburg
-					}
-				}
-			}
 			if = {
+				limit = { original_tag = TRD }
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/decisions/_generic_decisions.txt:81]: yep

File: common/decisions/_society_advancement_decisions.txt [hunk 38]
```diff
@@ -58,11 +56,8 @@ economy_decisions = {
 		}
 
 		visible = {
-			NOT = {
-				has_idea = modern_society
-			}
+			NOT = { has_idea = modern_society }
 			has_country_flag = can_improve_society_development
-			tag = NIM
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/decisions/_society_advancement_decisions.txt:65]: Check "allowed"

File: common/decisions/categories/SRS_decision_categories.txt [hunk 39]
```diff
@@ -1,12 +1,11 @@
 SRS_the_defence_of_skynavia = {
 	icon = eng_move_to_secure_the_dominions
 	allowed = {
-		original_tag = SRS
+		tag = SRS
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/decisions/categories/SRS_decision_categories.txt:4]: Reverted, I don't even know why I changed it.

File: common/technologies/infantry.txt (hunk 7 above)
Comment (yastrom) [reply common/technologies/infantry.txt:166]: Too big value turned into negative when multiplied by AI factors. I have no idea who might need it, but I will give it back if necessary.

File: common/scripted_triggers/EAW_triggers.txt (hunk 9 above)
Comment (yastrom) [reply common/scripted_triggers/EAW_triggers.txt:1394]: I added this a long time ago and subsequently refused as unnecessary. I haven't seen any examples of anyone using these triggers, so I decided to delete them.

File: common/national_focus/SRS.txt (hunk 27 above)
Comment (yastrom) [reply common/national_focus/SRS.txt:1917]: There is no need for this, these two checks are mutually exclusive, but for some reason both used IF (such errors are repeated many times, but I don’t want to correct them all).

File: common/on_actions/eaw_on_actions.txt (hunk 31 above)
Comment (yastrom) [reply common/on_actions/eaw_on_actions.txt:3411]: OK

File: common/on_actions/eaw_on_actions.txt (hunk 32 above)
Comment (yastrom) [reply common/on_actions/eaw_on_actions.txt:4291]: Git incorrectly selects the brackets, nothing was removed.

File: common/decisions/_generic_decisions.txt [hunk 40]
```diff
@@ -152,28 +145,41 @@ political_actions = {
...
+						}
 					}
 				}
 			}
+		}
+
+		visible = {
+			always = yes
 			if = {
+				limit = { original_tag = TRD }
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/decisions/_generic_decisions.txt:178]: fixed

File: common/technology_sharing/eaw_tech_sharing_groups.txt (hunk 6 above)
Comment (yastrom) [reply common/technology_sharing/eaw_tech_sharing_groups.txt:250]: It was swapped, although I do not know why.

File: common/modifiers/00_static_modifiers.txt (hunk 26 above)
Comment (yastrom) [reply common/modifiers/00_static_modifiers.txt:260]: probably mistake

File: common/opinion_modifiers/eaw_opinion_modifiers.txt (hunk 33 above)
Comment (yastrom) [reply common/opinion_modifiers/eaw_opinion_modifiers.txt:2]: Only duplicate modifiers was removed, nothing more.

File: common/continuous_focus/generic.txt [hunk 41]
```diff
@@ -1,66 +1,70 @@
...
 	
-		
 	focus = {
 		id = continuous_naval_production
-		
 		icon = GFX_goal_continuous_naval_production
 		
-		available = {
-		}
+		daily_cost = 1
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/continuous_focus/generic.txt:17]: Refactoring

File: common/continuous_focus/generic.txt [hunk 42]
```diff
@@ -73,61 +77,93 @@ continuous_focus_palette = {
...
 		}
-
+		
 		enable = {
 			has_completed_focus = MAN_beginning_2_1
 		}
-
-		modifier = {
-		}
+
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/continuous_focus/generic.txt:163]: The focus after the merge did not lose any functionality, it was only adapted to the general structure.

File: common/country_leader/00_traits.txt [hunk 43]
```diff
@@ -1309,34 +1304,41 @@ leader_traits = {
...
 	}
-
+	
 	incognito = {
 		fascism_drift = 0.1
 		drift_defence_factor = 0.15
 	}
-
+	
+	#child_of_blizzards = {
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/country_leader/00_traits.txt:1330]: already

File: common/country_leader/00_traits.txt [hunk 44]
```diff
@@ -1951,6 +1979,24 @@ leader_traits = {
 		}
 	}
 	
+	silent_workgriffon1 = {
```
[the remark this answers is not in the document]
Comment (yastrom) [reply common/country_leader/00_traits.txt:1982]: At first, the harmonic tribes are in a "bad" situation, Fiana's political advisors do not have much experience, but as the country develops, they will improve their skills.

File: history/states/567-Cubtria.txt [hunk 45]
```diff
@@ -1,9 +1,8 @@
 state = {
 	id = 567
 	name = "STATE_567"
-	manpower = 352500
-	
 	state_category = rural
+	manpower = 352500
```
[the remark this answers is not in the document]
Comment (yastrom) [reply history/states/567-Cubtria.txt:5]: What's wrong with that?
